pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -81,6 +83,33 @@ func writeVifFile(buf []byte, pid, name string) error {
 	return nil
 }
 
+func setPodInterfaceCache(iface *v1.Interface, podInterfaceName string, uid string) error {
+	link, err := Handler.LinkByName(podInterfaceName)
+	if err != nil {
+		log.Log.Reason(err).Errorf("failed to get a link for interface: %s", podInterfaceName)
+		return err
+	}
+	// get IP address
+	addrList, err := Handler.AddrList(link, netlink.FAMILY_V4)
```

Comment:
does this account for ipv6?

Reply:
Do you mean dual-stack? I missed the IPv6 here, are the vm pods allowed to have IPv6?
I tried `netlink.FAMILY_ALL` and it seemed to be working exactly the same way

Reply:
@AlonaKaplan can you weigh in here?

what we're attempting to do here is detect what the ip address is of the pod network interface from the pod's network namespace. For masquerade, this IP address will ultimately be the one set on the VMI's status.

Since masquerade supports IPv6 now, should be we using netlink.FAMILY_ALL here?

Reply:
Currently we are not supporting dual stack. For dual stack we will need to add both ipv4 and ipv6 addresses to the status.
For single stack we can use netlink.FAMILY_ALL or use Handler.IsIpv6Enabled() to decide what address family should be used.

Reply:
Lane pull-kubevirt-e2e-kind-k8s-1.17.0-ipv6  should have caught the bug (reading the ipv4 address and not ipv6).
But seems its totally broken on this PR (I don't know if it is the PRs fault or another problem).

Reply:
`netlink.FAMILY_ALL` seems simpler approach here, changed to it now